### PR TITLE
Create cookbook example page(s) for grid layout

### LINKED ISSUE #1668: [Enhancement] Cookbook example page for initial grid layout example
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Create a cookbook example, that can later be used in a popout example on a cookbook page. The example should show the grid layout from #1521. 

## Describe the solution you'd like
Create a cookbook example ([like this one](https://cookbook.kirby.design/examples/accordion)) that shows the layout from https://codepen.io/jkaltoft/pen/gOWPyqv, with a couple of kirby cards or similar as content. Then consumers can inspect this, until we have created a dedicated cookbook page in #1669 

## Have you considered any alternatives?
No, we need this simple example before creating best practices for layouts etc. in the cookbook.

## Are there any additional context?
See #1521 for context and screenshots of the intended functionality. 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](../CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](../CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [x] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by jkaltoft**
## Which issue does this PR close?

This PR closes #1668

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](./CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).

### CODE REVIEW

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]
Really great examples of grid usage. Will be even more awesome combined with a showcase down the line!

**Review by RasmusKjeldgaard — APPROVED** [review]
LGTM... Because I made the changes!

File: apps/cookbook/src/app/examples/grid-layout-example/grid-layout-core-example/grid-layout-core-example.component.scss [hunk 1]
```diff
@@ -1,10 +1,8 @@
+@import './libs/designsystem/src/lib/scss/utils';
+
 // Grid
 $columns: 12;
```
Comment (MadsBuchmann) [inline apps/cookbook/src/app/examples/grid-layout-example/grid-layout-core-example/grid-layout-core-example.component.scss:4]: We could consider if we should have specific design tokens for grid in `_variables.scss`. 

But perhaps this should be talked more over and a new issue should be created for this

**Review by MadsBuchmann — APPROVED** [review]
Ship it!

**Pull request merged**